projects/admin/src/app/circulation/patron/main/main.component.html

```diff
@@ -39,32 +39,38 @@
           class="nav-link"
           routerLinkActive="active"
           [routerLinkActiveOptions]="{exact: true}"
-          [routerLink]="['/circulation', 'patron', patron.patron.barcode, 'loan']"
-          translate
-          >Checkin/Checkout</a
-        >
+          [routerLink]="['/circulation', 'patron', patron.patron.barcode, 'loan']">
+          {{ 'Checkin/Checkout' | translate }}
+          <span *ngIf="this.patronCirculationStatistics.loans > 0" class="badge badge-info font-weight-normal">
```

Comment:
Why do you hide the badge if the result is 0? The user will not click if the total is 0